graphs/digraph.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
This is a  Unused import os
```suggestion 
```

 from our Suggestion Bot